proxies/venue-graphql-proxy/src/integrations/VenueLinkedIntegration.ts

```diff
@@ -0,0 +1,96 @@
+import type VenueEnricher from '../enrichers/VenueEnricher';
+import type { TranslationsObject, VenueDetails } from '../types';
+import { translateVenue } from '../utils/utils';
+import VenueResolverIntegration from './VenueResolverIntegration';
+
+type LinkedTranslationObject = {
+  fi: string | null;
+  sv: string | null;
+  en: string | null;
+} | null;
+
+type LinkedDivision = {
+  type: string;
+  ocd_id: string;
+  municipality: string | null;
+  // Seems that this translation may behave differently than the rest.
+  name: {
+    fi: string;
+    sv?: string;
+    en?: string;
+  };
+};
+
+type LinkedPoint = {
+  type: 'Point';
+  coordinates: [number, number];
+};
+
+type LinkedPlace = {
+  id: string;
+  data_source: string | null;
+  publisher: string | null;
+  divisions: LinkedDivision[];
+  created_time: string;
+  last_modified_time: string;
+  custom_data: unknown | null;
+  email: string | null;
+  address_region: string | null;
+  postal_code: string | null;
+  post_office_box_num: string | null;
+  address_country: string | null;
+  deleted: boolean;
+  has_upcoming_events: boolean;
+  n_events: number;
+  image: string | null;
+  parent: string | null;
+  replaced_by: string | null;
+  position: LinkedPoint;
+  address_locality: LinkedTranslationObject;
+  description: LinkedTranslationObject;
+  name: LinkedTranslationObject;
+  info_url: LinkedTranslationObject;
+  street_address: LinkedTranslationObject;
+  telephone: LinkedTranslationObject;
+  '@id': string;
+  '@context': string;
+  '@type': string;
+};
+
+type Config<I, O> = {
+  enrichers?: VenueEnricher<I, O>[];
+};
+
+export default class VenueLinkedIntegration<
```

Comment:
The PR is so huge that I just wanted to write this down, so it will be investigated a bit...

I think this whole LinkedEvents integration could be deprecated. I don't think we have any use for it at the moment, since we have another proxy-server for the LinkedEvents connection. This was in the old Liikunta, where there were no other LinkedEvent connections.

Reply:
Perhaps I would just remove it. It can be retained from old liikunta repo? Also these files still exist under apps/events-helsinki-sports because I was planning to remove them in next pr where sports start to use the venue-graphql-proxy